lib/local_daily_digest.js

```diff
@@ -30,18 +39,61 @@ function get(masterCb) {
 				lastDigestSent = new Date(lastDigestSent);
 				var offset = current.valueOf() - lastDigestSent.valueOf();
 				if (offset > 24 * 60 * 60 * 1000) { // If last digest was sent more than 24 hours ago
+					//if profile roles does not contain manager or editor
 	  				allLocalProfilesWDigest.push(profile);
+	  			
+	  				if(!(matchInArray('manager:', profile.roles)) && !(matchInArray('editor:', profile.roles)))
```

Comment:
Oh it should be if profile.roles does not have Manager OR profile.roles has editor I believe. It should not be an AND conition